Geometry/HGCalCommonData/plugins/DDHGCalEEAlgo.cc

```diff
@@ -26,7 +26,7 @@
 #include <unordered_set>
 #include <vector>
 
-//#define EDM_ML_DEBUG
+#define EDM_ML_DEBUG
```

Comment:
Are the debug messages really needed in the master?

Reply:
@bsunanda 
Could you please confirm?

Reply:
Sorry I am correcting for that